ETSI/Python/_py__NFVI_Resources_Management/Process_Delete_VIM/Tasks/Task_Delete_VIM_Resources.py

```diff
@@ -16,5 +16,5 @@
     
     r = nfviVim.nfvi_vim_delete(context["vim_id"])
     
-    ret = MSA_API.process_content('ENDED', f'{r}', context, True)
+    ret = MSA_API.process_content(f'{nfviVim.state}', f'{r}', context, True)
```

Comment:
Shouldn't it be **nfviVim.state** instead of **f'{nfviVim.state}'**

Reply:
yes, thank you for spotting this